src/PowerJoinClause.php

```diff
@@ -13,6 +14,20 @@ class PowerJoinClause extends JoinClause
      */
     public $model;
 
+    /**
+     * Table name backup in case an alias is being used.
+     *
+     * @var string
+     */
+    public $tableName;
```

Comment:
@pddevins Yeah, is needed because when we add an alias, `$this->table` becomes `categories as {alias}`, and we need to replace the table name with the alias in the join clauses later on.